$NetBSD: patch-gcc_doc_invoke.texi,v 1.1 2013/11/16 07:41:00 mef Exp $

wip/nios2-gcc/work/gcc-4.1/gcc/doc/invoke.texi:1078: I can't find file `'.

--- gcc/doc/invoke.texi.orig	2013-06-14 06:32:55.000000000 +0900
+++ gcc/doc/invoke.texi	2013-11-09 10:31:35.000000000 +0900
@@ -1075,7 +1075,7 @@
 @opindex version
 Display the version number and copyrights of the invoked GCC@.
 
-@include @value{srcdir}/../libiberty/at-file.texi
+@c @include @value{srcdir}/../libiberty/at-file.texi
 @end table
 
 @node Invoking G++
